<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_CADRE</class>
 <widget class="QMainWindow" name="C_CADRE">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>C_CADRE</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="verticalSlider1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>16</width>
      <height>160</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Box_nombre1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>320</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>290</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>290</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Min</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>320</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>max</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>C_CADRE</receiver>
   <slot>Slot_Get_Min(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Box_nombre1</sender>
   <signal>textChanged(QString)</signal>
   <receiver>C_CADRE</receiver>
   <slot>Slot_Get_Max(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>353</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>370</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>C_CADRE</receiver>
   <slot>Slot_Recuperer_Valeur_Slider(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Slot_Get_Min(QString)</slot>
  <slot>Slot_Get_Max(QString)</slot>
  <slot>Slot_Recuperer_Valeur_Slider(int)</slot>
 </slots>
</ui>
